login-workflow/src/screens/refactoredSubscreens/ExistingAccountRegisterSuccess.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { useLanguageLocale } from '@brightlayer-ui/react-auth-shared';
+import { FinishState } from '../../components';
+import Person from '@mui/icons-material/Person';
+import { BrandedCardContainer, BrandedCardContainerProps } from '../../components/BrandedCardContainer';
+import Divider from '@mui/material/Divider';
+import CardHeader from '@mui/material/CardHeader';
+import Typography from '@mui/material/Typography';
+import { DialogActionsStyles, DialogButtonStyles, DialogContentStyles, DialogTitleStyles } from '../../styles';
+import { useTheme } from '@mui/material/styles';
+import CardContent from '@mui/material/CardContent';
+import CardActions from '@mui/material/CardActions';
+import Button from '@mui/material/Button';
+import { useRoutes } from '../../contexts/RoutingContext';
+import { useNavigate } from 'react-router-dom';
+
+/**
+ * Component that renders a screen displaying success for creating an account for
+ * a user that already exists.
+ *
+ * @category Component
+ */
+
+type ExistingAccountRegisterSuccessProps = BrandedCardContainerProps & {
+    showInCard?: boolean;
+    cardTitle?: string;
+    messageIcon?: JSX.Element;
+    messageTitle?: string;
+    messageBody?: string;
+    actions?: JSX.Element | JSX.Element[];
+};
+
+export const ExistingAccountRegisterSuccess: React.FC<ExistingAccountRegisterSuccessProps> = (props) => {
```

Comment:
Actually `ExistingAccountComplete` component was a Subscreen and it was just rendering message part inside the `CardContent` component. As part of the story have added `CardHeader`, `CardContent` and `CardActions` components wrapped by `BrandedCardContainer` and now it is rendering a complete screen so adding SubScreen in the name doesn't make sense.

Reply:
We (you, me & Tom) discussed this during PM office hours regarding whether to create a screen or a subscreen and reached the conclusion that it is better to create subscreens than screens. You did not raise any concerns about this. Please justify or invite Joe to the conversation next time.

I still see that it is more beneficial to create subscreens rather than screens.

Reply:
Yes Okay, will rename as `ExistingAccountRegisterSuccessSubscreen`. Thanks